media-api/app/controllers/collections.txt

```diff
@@ -0,0 +1,51 @@
+cities
+family
+filmandmusic
+lr
+pp
+saturdayreview
+trv
```

Comment:
I'd argue they're not cryptic - just lingo. Either way - I think it might help if we find a slightly nicer way to display them i.e. not massive blue labels on each image.